apps/web/src/app/vault/individual-vault/vault.component.ts

```diff
@@ -465,7 +465,11 @@ export class VaultComponent implements OnInit, OnDestroy {
         switchMap(() =>
           combineLatest([
             filter$,
-            this.billingAccountProfileStateService.hasPremiumFromAnySource$,
+            this.accountService.activeAccount$.pipe(
+              switchMap((account) =>
+                this.billingAccountProfileStateService.hasPremiumFromAnySource$(account.id),
+              ),
+            ),
```

Comment:
I think this can be simplified, we have activeUserId initialized on L288

```suggestion
            this.billingAccountProfileStateService.hasPremiumFromAnySource$(this.activeUserId)
```